apps/ui/src/views/Space/Discussions.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { loadTopics, SPACES_DISCUSSIONS, Topic } from '@/helpers/discourse';
+import { Space } from '@/types';
+import ICDiscourse from '~icons/c/discourse';
+
+const props = defineProps<{ space: Space }>();
+
+const { setTitle } = useTitle();
+
+const topics: Ref<Topic[]> = ref([]);
+const loading = ref(false);
+
+const discussionsUrl =
+  SPACES_DISCUSSIONS[`${props.space.network}:${props.space.id}`];
+
+onMounted(async () => {
+  try {
```

Comment:
Not a big deal as this page shouldn't be accessible without `discussionsUrl` but I'd just add check to see if `discussionsUrl` is defined before calling it.